src/resources/views/crud/fields/fetch.blade.php

```diff
@@ -0,0 +1,212 @@
+@php
+
+    $routeEntity = strtolower(preg_replace('/([a-zA-Z])(?=[A-Z])/', '$1-', $field['entity']));
+    $connected_entity = new $field['model'];
+
+    $connected_entity_key_name = $connected_entity->getKeyName();
+    $old_value = old(square_brackets_to_dots($field['name'])) ?? $field['value'] ?? $field['default'] ?? false;
+
+    $response_entity = isset($field['response_entity']) ? $field['response_entity'] : $crud->hasOperationSetting('ajaxEntities') ? array_has($crud->getOperationSetting('ajaxEntities'), $field['entity']) ? $field['entity'] : array_key_first($crud->getOperationSetting('ajaxEntities')) : '';
```

Comment:
Hmm... I'm not sure I understand how the developer would use a setting like ```ajaxEntities``` for the Fetch operation. Where would it mostly be used, inside the ```crud.php``` config file or inside each ```CrudController```? 

Could you please explain the benefit of this setting?

Reply:
That is all changed as per your first review. We no longer use crud settings or anything for that. just the plain functions. I need to push my last changes that will reflect that, i am finishing implementing the multiple functionality, atm i have only changed for single select. 

Give me an hour or so and i will push my local branch here, then i think you can make your final review.